tests/Samples.Server.Tests/ResponseTests.cs

```diff
@@ -115,14 +116,23 @@ public async Task Batched_Query_Should_Return_Multiple_Results()
                 "JSON body text could not be parsed. 'o' is an invalid start of a property name. Expected a '\"'. Path: $ | LineNumber: 0 | BytePositionInLine: 1."
             },
 
+            // POST with JSON mime type that is null JSON should be a bad request
+            new object[]
+            {
+                HttpMethod.Post,
+                new StringContent("null", Encoding.UTF8, "application/json"),
+                HttpStatusCode.BadRequest,
+                "GraphQL query is missing."
+            },
```

Comment:
I added an additional test here in case the string `"null"` was passed in, mainly to be sure there is no null reference exception.  Note that the string `"{}"` would have the same effect.